cli/mqtt.py

```diff
@@ -12,6 +14,8 @@
 
 
 def mqtt_open(config, insecure):
+    cert_path = path.join(ROOT_DIR, "ssl/ankermake-mqtt.crt")
```

Comment:
No need to import `os.path`:

```python
cert_path = ROOT_DIR / "ssl/ankermake-mqtt.crt"
```

(that's the new Pathlib.Path syntax)

Reply:
✅